src/domain/position/binProcessor.js

```diff
@@ -88,6 +89,15 @@ const processBins = async (bins, trx) => {
     Enum.Accounts.LedgerEntryType.PRINCIPLE_VALUE
   )
 
+  // Fetch all RESERVED participantPositionChanges associated with a commitRequestId
+  // These will contain the value that was reserved for the fxTransfer
+  // We will use these values to revert the position on timeouts
+  const fetchedReservedPositionChangesByCommitRequestIds =
+    await BatchPositionModel.getReservedPositionChangesByCommitRequestIds(
+      trx,
+      commitRequestIdList
+    )
```

Comment:
Do we need to get position changes for all the accountIds? Or do you want to pass the accountId and query only for that accountId?

Reply:
We model a lot of our reads to just do one select so we have all the data at hand in the bin processor, no?
Querying specific accountId's one at a time would defeat the purpose.

Reply:
Oh you are right. I thought we only work on single accountId in the bin processor.